hr_skill/views/hr_employee.xml

```diff
@@ -8,7 +8,14 @@
         <field name="arch" type="xml">
             <notebook position="inside">
                 <page string="Skills">
-                    <field name="skill_ids" />
+                    <field name="skill_ids" domain="[('state', '=', 'approved')]">
+                        <tree editable="top">
+                            <field name="display_name" readonly="1"/>
+                            <field name="state" readonly="1"/>
+                            <field name="free_text" attrs="{'readonly': [('is_free_text_field', '=', False)]}"/>
+                            <field name="is_free_text_field" invisible="1"/>
+                        </tree>
```

Comment:
Here also, could you check if the effect is the same if this inline view is removed here and instead you change [the generic tree view for `hr.skill`](https://github.com/OCA/hr/blob/8.0/hr_skill/views/hr_skill.xml#L10)?

If you want to have some deviations in that one, you can create a new tree view for that, assign a high priority (eg. 101) and reference that one explicitly using `view_id` in [the action](https://github.com/OCA/hr/blob/8.0/hr_skill/views/hr_skill.xml#L57).